packages/sync-repo-settings/src/sync-repo-settings.ts

```diff
@@ -193,17 +206,34 @@ async function updateRepoTeams(repo: Repo, context: Context) {
   console.log(`Update team access for ${repo.repo}`);
   const [owner, name] = repo.repo.split('/');
   const teamsToAdd = getRepoTeams(repo.language);
-  await Promise.all(
-    teamsToAdd.map(membership => {
-      return context.github.teams.addOrUpdateRepoInOrg({
-        team_slug: membership.slug,
-        owner,
-        org: owner,
-        permission: membership.permission as 'push',
-        repo: name,
-      });
-    })
-  );
+  try {
+    await Promise.all(
+      teamsToAdd.map(membership => {
+        return context.github.teams.addOrUpdateRepoInOrg({
```

Comment:
I think this might actually be ```addOrUpdateRepoPermissionsInOrg``` (https://octokit.github.io/rest.js/v18#teams-add-or-update-repo-permissions-in-org)